### PR TITLE
[chore] update schedule, fix couple of typos

### PR DESCRIPTION
**Pull request by codeboten**

### CODE REVIEW

File: docs/release.md [hunk 1]
```diff
@@ -27,13 +27,13 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 1. Update Contrib to use the latest in development version of Core. Run `make update-otel` in Contrib root directory and if it results in any changes submit a draft PR to Contrib. Ensure the CI passes before proceeding. This is to ensure that the latest core does not break contrib in any way. We’ll update it once more to the final release number later.
 
-2. Determine the version number that will be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
+2. Determine the version number that wil be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
```
Comment (atoulme) [inline docs/release.md:30]: This adds a typo?

Comment (codeboten) [reply docs/release.md:30]: omg... thanks i added it to test spellcheck lol and left it there 🤦🏻

Comment (codeboten) [inline docs/release.md:30]:
```suggestion
2. Determine the version number that will be assigned to the release. Usually, we increment the minor version number and set the patch number to 0. In this document, we are using `v0.85.0` as the version to be released, following `v0.84.0`.
```

Comment (codeboten) [reply docs/release.md]: @atoulme fixed.... although you know fixing 3 typos adding 1, technically still making things better 😆

Comment (atoulme) [reply docs/release.md]: Good running average

File: docs/release.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ It is possible that a core approver isn't a contrib approver. In that case, the
...
 
-3. Manually run the action [Automation - Prepare Release](https://github.com/open-telemetry/opentelemetry-collector/actions/workflows/prepare-release.yml). This action will create an issue to track the progress of the release and a pull request to update the changelog and version numbers in the repo. **While this PR is open all merging in Core should be haulted**.
+3. Manually run the action [Automation - Prepare Release](https://github.com/open-telemetry/opentelemetry-collector/actions/workflows/prepare-release.yml). This action will create an issue to track the progress of the release and a pull request to update the changelog and version numbers in the repo. **While this PR is open all merging in Core should be halted**.
    - When prompted, enter the version numbers determined in Step 2, but do not include a leading `v`.
-   - If not intending to release stable modeles, do not specify a version for `Release candidate version stable`.
+   - If not intending to release stable models, do not specify a version for `Release candidate version stable`.
```
Comment (mx-psi) [inline docs/release.md:36]: I guess this was meant to be 'modules'?
```suggestion
   - If not intending to release stable modules, do not specify a version for `Release candidate version stable`.
```

File: docs/release.md [hunk 3]
```diff
@@ -59,7 +59,7 @@ It is possible that a core approver isn't a contrib approver. In that case, the
 
 ## Releasing opentelemetry-collector-contrib
 
-1. Manually run the action [Automation - Prepare Release](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/workflows/prepare-release.yml). When prompted, enter the version numbers determined in Step 1, but do not include a leading `v`. This action will a pull request to update the changelog and version numbers in the repo. **While this PR is open all merging in Contrib should be haulted**.
+1. Manually run the action [Automation - Prepare Release](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/workflows/prepare-release.yml). When prompted, enter the version numbers determined in Step 1, but do not include a leading `v`. This action will a pull request to update the changelog and version numbers in the repo. **While this PR is open all merging in Contrib should be halted**.
```
Comment (crobert-1) [inline docs/release.md:62]:
```suggestion
1. Manually run the action [Automation - Prepare Release](https://github.com/open-telemetry/opentelemetry-collector-contrib/actions/workflows/prepare-release.yml). When prompted, enter the version numbers determined in Step 1, but do not include a leading `v`. This action will create a pull request to update the changelog and version numbers in the repo. **While this PR is open all merging in Contrib should be halted**.
```

**Review by crobert-1 — CHANGES_REQUESTED** [review]

**Review by crobert-1 — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Pull request merged**